doc/pages/administrator/070-manage_realms.md

```diff
@@ -13,6 +13,50 @@ kubectl get secret -n astarte astarte-housekeeping-private-key -o=jsonpath={.dat
 
 You can then use `housekeeping.key` to authenticate against Housekeeping API.
 
+## Device limit in a Realm
+While there is no limit on the number of Devices registered in a Realm, it is possible to impose
+an upper bound using the `<astarte base API URL>/housekeeping/v1/realms/<realm name>` API.
+By default, such an upper bound does not exist. The limit can be retrieved, updated or removed
+(meaning that any number of registered Devices is allowed).
+
+### Setting, updating or removing the limit on the number of registered Devices in a Realm
+
+`PATCH <astarte base API URL>/housekeeping/v1/realms/<realm name>`
+
+The HTTP payload of the request must have the following format:
+
+```json
+{
+    "data": {
+        "device_registration_limit": <value>
+    }
+}
+```
+
+To set or update the limit, the value must be a non negative integer.
+If the value is less than the number of currently registered devices the HTTP call will fail.
```

Comment:
IIRC this is not the actual behavior anymore

Reply:
Good catch